adwords_client/adwords_api/operations/attach_label.py

```diff
@@ -25,9 +23,13 @@ def attach_label_operation(label_id: 'Long' = None,
         operation['operand']['adGroupId'] = ad_group_id
         operation['xsi_type'] = 'AdGroupLabelOperation'
     if ad_id:
+        if campaign_id:
+            operation['operand'].pop('campaignId', None)
```

Comment:
I believe the check for `ad_id` should come before the one for `campaign_id` and do the "right" thing straight away instead of "undoing" what the `if campaign_id:` did.